j/jquery-labelauty/jquery-labelauty_v1.1.4_ubi8.5.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package: jquery-labelauty
+# Version: v1.1.4
+# Source repo: https://github.com/fntneves/jquery-labelauty 
+# Tested on: RHEL v8.5
```

Comment:
can we make this "UBI 8.5" for consistency with the build script name?

Reply:
done